.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
 /build
 /dist
+/venv
```

Comment:
No need to put this here. The virtualenv folder doesn't have to live inside the code repository and it doesn't always have that name. For instance I'm using virtualenvwrapper, so all of my virtualenvs are in one place.

But if you want to ignore it, you can just add it to your local `.git/info/exclude`

Reply:
My bad. Didn't mean to push this. I've gotten in the habbit of making a venv in most of my python projects (but I'm not really a python guy). I didn't know you can make a local exclude for a repo. That's neat :)

I could also probably just make a global exclude for this. Will update later this morning!